spec/models/time_profile_spec.rb

```diff
@@ -11,6 +11,103 @@
     expect(t.tz).to be_nil
   end
 
+  describe "Test profile= with days" do
+    before do
+      @tp = TimeProfile.new
+      @tp.profile = data
+    end
+    context "with symbol keys" do
+      let(:data) { {:days => [0, 1, 2, 3]} }
+      it "returns the correct days" do
+        expect(@tp.days).to eq([0, 1, 2, 3])
+      end
+    end
+    context "with string keys" do
+      let(:data) { {"days" => [0, 1, 2, 3]} }
+      it "returns the correct days" do
+        expect(@tp.days).to eq([0, 1, 2, 3])
+      end
+    end
+  end
+
+  describe "Test profile= with hours" do
+    before do
+      @tp = TimeProfile.new
+      @tp.profile = data
+    end
+    context "with symbol keys" do
+      let(:data) { {:hours => [0, 1, 2, 3]} }
+      it "returns the correct days" do
+        expect(@tp.hours).to eq([0, 1, 2, 3])
+      end
+    end
+    context "with string keys" do
+      let(:data) { {"hours" => [0, 1, 2, 3]} }
+      it "returns the correct days" do
+        expect(@tp.hours).to eq([0, 1, 2, 3])
+      end
+    end
+  end
+
+  describe "Test profile=" do
+    before do
+      @tp = TimeProfile.new
+      @tp.profile = data
+    end
+    context "with symbol keys" do
+      let(:data) {
```

Comment:
A multi-line block is typically:
```
let(:data) do
  {
  }
end
```

Reply:
Also what rubocop said ☝️ https://github.com/ManageIQ/manageiq/pull/21311#issuecomment-871414363